js/script.js

```diff
@@ -0,0 +1,112 @@
+window.onload = () => {
+  const guestSpeakers = [
+    {
+      image: './assets/images/toto.png',
+      name: 'Torger Christian Wolff',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/checop.png',
+      name: 'Sergio Michel Pérez Mendoza',
+      role: 'f1 driver',
+      bio: 'Redbull Racing Driver',
+    },
+    {
+      image: './assets/images/lewish.png',
+      name: 'Lewis Carl Davidson Larbalestier Hamilton',
+      role: 'f1 driver',
+      bio: 'Mercedes AMG Petronas Motorsport',
+    },
+    {
+      image: './assets/images/christianh.png',
+      name: 'Christian Horner',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/zacb.png',
+      name: 'Zak Brown',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+    {
+      image: './assets/images/binnoto.png',
+      name: 'Mattia Binotto',
+      role: 'Formula 1 manager',
+      bio: 'Shareholder and director in Formula 1',
+    },
+  ];
+
+  
+    const guestSpeakerAtEvent = document.getElementById('image-display');
+    for (let i = 0; i < guestSpeakers.length; i++){
+      let speakers = `<div id="card${i}" class="speaker-1">
+    <div class="speaker-image">
+        <img src=${guestSpeakers[i].image} alt="speaker-1">
+    </div>
+    <div>
+        <h2>${guestSpeakers[i].name}</h2>
+        <h3>${guestSpeakers[i].role}</h3>
+        <h4>${guestSpeakers[i].bio}</h4>
+    </div>
+</div>`;
+guestSpeakerAtEvent.innerHTML+=speakers;
+console.log(speakers);
```

Comment:
## Optional:
- [ ] As previous reviewer didn't mention it but it's a project requirement this is highly important. Your speakers section just works fine on mobile, however when I go desktop, I can't see the 6 speakers in the grids as 3 columns for each row: 
![image](https://user-images.githubusercontent.com/91486791/205137948-99a02ed7-3f8a-4776-bb70-6fca750dbc05.png)
![image](https://user-images.githubusercontent.com/91486791/205138023-686ee7dc-9654-43bc-9d39-58069ebf0325.png)